plugins/modules/zypper.py

```diff
@@ -158,6 +158,13 @@
     description:
       - Adds C(--quiet) option to I(zypper) install/update command.
     version_added: '10.2.0'
+  skip_post_errors:
+    type: bool
+    required: false
```

Comment:
Not a _problem_ but we usually omit `required=false` from the parameters, that is already implicit. Feel free to keep it or remove it, this is not a show-stopper.